### DISCUSSION TITLE
Error: Undefined mixin. - @include u-font-family()

### DISCUSSION THREAD

**Discussion by mayelaatf** [uswds/uswds#4666] [General]
Hi,

I installed uswds 3.0.0 , and I am using @uswds/compile package to compile the sass.
Added to my gulpfile.js the uswds version : 3.0  and updated the paths to go to @uswds/uswds

My problem is that after I compile SASS I get this :
Error: Undefined mixin.

When I am using @include u-font-family('serif');

Why am I missing?

Mayela

**Comment by mayelaatf** [comment]
I figured it out - It needs this to load “uswds-core” at the top of the stylesheet that uses USWDS members (tokens, variables, mixins, functions, or placeholders) 
@use "uswds-core" as *;

**Reply by mejiaj** [reply]
Glad you got it sorted 😃
